integrations/model-training/pytorch/pytorch-rich-logging/README.md

```diff
@@ -0,0 +1,32 @@
+# Pytorch integration with Comet.ml
+
+Comet integrates with [PyTorch](https://pytorch.org/).
+
+PyTorch is an open source machine learning framework based on the Torch library, used for applications such as computer vision and natural language processing, primarily developed by Facebook's AI Research lab.
+
+PyTorch enables fast, flexible experimentation and efficient production through a user-friendly front-end, distributed training, and ecosystem of tools and libraries.
```

Comment:
[PyTorch](https://github.com/pytorch/pytorch) is a popular open source machine learning framework based on the Torch library, used for applications such as computer vision and natural language processing.

PyTorch enables fast, flexible experimentation and efficient production through a user-friendly front-end, distributed training, and ecosystem of tools and libraries.

Instrument PyTorch with Comet to start managing experiments, create dataset versions and track hyperparameters for faster and easier reproducibility and collaboration.